migrations/2023-05-05-112013_add_evidence_col_in_dispute/up.sql

```diff
@@ -0,0 +1,3 @@
+-- Your SQL goes here
+ALTER TABLE dispute
+ADD COLUMN evidence JSONB NOT NULL DEFAULT '{}'::JSONB;
```

Comment:
Remove not null constraints

Reply:
It's backward compatible as there is no dispute record yet!